docs/assistant/security-assistant.asciidoc

```diff
@@ -168,7 +168,7 @@ The *Show anonymized* toggle controls whether you see the obfuscated or plaintex
 
 * **Knowledge base:** Use retrieval-augmented generation to provide specialized knowledge of the Elastic Search Query Language ({esql}) to AI Assistant. For example, with the knowledge base active, you can ask AI Assistant to help you write an {esql} query for a particular use case, or ask it to answer general questions about {esql} syntax and usage. Without the knowledge base enabled, AI Assistant will not be able to answer questions about {esql}.
 +
-beta::[]
+beta::["Do not use {esql} on production environments. This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
```

Comment:
I suggest using the "beta" text instead of the "technical preview" text, as the `beta` admonition is used here. (I guess the AI assistant is in beta instead of tech preview?)
```suggestion
beta::["Do not use {esql} on production environments. This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features."]
```

Reply:
Yeah, makes sense. @benironside would you mind confirming that the AI Assistant knowledge base feature is in the beta release stage?

Reply:
Looks like you might've removed it [here](https://github.com/elastic/security-docs/pull/4520/files#diff-1100fa48f70df582d608ab5c60d2101aaaba89cdd2b7d55b7de7d9658e90e6deR181), @benironside. I'll resolve this change for now since it'll be overwritten anyways once you merge your PR.